cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -123,22 +123,18 @@ Slice::computeDefaultSerialVersionUID(const ContainedPtr& p)
 
     // Actually compute the `SerialVersionUID` value.
     ostringstream os;
-    os << "Name: " << name;
+    os << name << ":";
     if (baseName)
     {
-        os << " Base: [" << *baseName << "]";
+        os << *baseName;
     }
-    os << " Members: [";
-    for (DataMemberList::const_iterator i = members.begin(); i != members.end();)
+    os << ";";
+    for (const auto& member : members)
     {
-        os << (*i)->name() << ":" << (*i)->type();
-        i++;
-        if (i != members.end())
-        {
-            os << ", ";
-        }
+        const MetadataList metadata = member->getMetadata();
+        const string typeString = JavaGenerator::typeToString(member->type(), TypeModeMember, "", metadata);
+        os << member->name() << ":" << typeString << ",";
```

Comment:
This will result in an "extra" comma, which is fine.

Reply:
Yes, I know it's a little lame, but it's just getting fed into a hash function.
So I didn't want to waste time and clock cycles checking if it's the last member.
*shrugs*